tests/test_retry.py

```diff
@@ -76,7 +76,7 @@ def request_callback(request):
             if attempts_count[0] <= 2:
                 raise ConnectionError("Connection aborted.", BadStatusLine("''"))
             if attempts_count[0] == 3:
-                return (429, {}, {})
+                return (429, {}, u'')
```

Comment:
It was supposed to return a string rather than a dict, like here [1]. Python 3 wouldn't accept any type other than `str`.

[1] https://github.com/scrapinghub/python-hubstorage/blob/6098e59db1ad9bd04a60b650c4797d2cd7437f10/hubstorage/serialization.py#L17